clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonServerGroup.groovy

```diff
@@ -37,6 +38,15 @@ class AmazonServerGroup implements ServerGroup, Serializable {
   Map<String, Object> image
   Map<String, Object> launchConfig
   Map<String, Object> launchTemplate
+
+  /**
+   * The fields marked @Alpha are subject to change in the future due to addition of newer ASG features like multiple launch templates.
+   */
+  @Alpha
```

Comment:
marked as `Alpha` as the data model will probably change in the future when multiple launch templates support is added.